tools/etl/tg-jdbc-driver/tg-jdbc-driver/src/main/java/com/tigergraph/jdbc/restpp/RestppConnection.java

```diff
@@ -411,13 +411,15 @@ public RestppResponse executeQueries(final List<QueryParser> queries) throws SQL
    */
   private void getToken() throws SQLException {
     StringBuilder urlSb = new StringBuilder();
-    // /gsqlserver/gsql/authtoken is deprecated, it won't return error messages when
-    // password is wrong
-    urlSb.append("/restpp/requesttoken");
     // If restpp version is under 3.5, pass graph name as a parameter
     if (this.restpp_version.compareTo(new ComparableVersion("3.5.0")) < 0) {
+      // /gsqlserver/gsql/authtoken is deprecated, it won't return error messages when
+      // password is wrong
+      urlSb.append("/gsqlserver/gsql/authtoken");
```

Comment:
Is `/restpp/requesttoken` incompatible with 3.5-? I have tested `/gsqlserver/gsql/authtoken` with 3.4 and found it won't return error messages for wrong password.

Reply:
USBank uses v3.2.2, which doesn't support `/requesttoken`